src/helper.jl

```diff
@@ -90,8 +87,25 @@ function atan2k(y::Float64, x::Float64)
     end
     s = y/x
     t = s*s
-    # sleef does not use mla here!
-    u = @horner t c1 c2 c3 c4 c5 c6 c7 c8 c9 c10 c11 c12 c13 c14 c15 c16 c17 c18 c19
+    u = c19
```

Comment:
while sleef doesn't use horner here. I tested with and without horner and for the functions that depend on atank2 it made not difference in accuracy, nonetheless I reverted back to what sleef had (no horner).